------------------------------------------------------------------------
r12012 | mortenmacfly | 2020-03-31 17:44:13 +0200 (Tue, 31 Mar 2020) | 2 lines

* keybinder: remove old-school #pragmas to fix bug #936 and https://bugzilla.redhat.com/show_bug.cgi?id=1819154
- keybinder: also, remove Borland c related pragmas. (Who will ever compile C::B using Borland C?)

Index: src/plugins/contrib/keybinder/cbKeyConfigPanel.h
===================================================================
--- a/src/plugins/contrib/keybinder/cbKeyConfigPanel.h	(revision 12011)
+++ b/src/plugins/contrib/keybinder/cbKeyConfigPanel.h	(revision 12012)
@@ -10,10 +10,6 @@
 #ifndef CBKEYCONFIGPANEL_H
 #define CBKEYCONFIGPANEL_H
 
-#ifdef __BORLANDC__
-	#pragma hdrstop
-#endif
-
 // Modified Keybinder for CodeBlocks KeyBnder v2.0
 
 // ----------------------------------------------------------------------------
Index: src/plugins/contrib/keybinder/cbkeyConfigPanel.cpp
===================================================================
--- a/src/plugins/contrib/keybinder/cbkeyConfigPanel.cpp	(revision 12011)
+++ b/src/plugins/contrib/keybinder/cbkeyConfigPanel.cpp	(revision 12012)
@@ -12,10 +12,6 @@
 
 // Modified CodeBlocks KeyBnder v2.0 2019/04/8
 
-#if defined(__GNUG__) && !defined(__APPLE__)
-    #pragma implementation "cbkeybinder.h"
-#endif
-
 #include <vector>
 
 #include <sdk.h>
Index: src/plugins/contrib/keybinder/cbkeybinder.cpp
===================================================================
--- a/src/plugins/contrib/keybinder/cbkeybinder.cpp	(revision 12011)
+++ b/src/plugins/contrib/keybinder/cbkeybinder.cpp	(revision 12012)
@@ -12,10 +12,6 @@
 
 // Modified CodeBlocks KeyBnder v2.0 2019/04/8
 
-#if defined(__GNUG__) && !defined(__APPLE__)
-    #pragma implementation "cbkeybinder.h"
-#endif
-
 #include <vector>
 
 #include <sdk.h>
Index: src/plugins/contrib/keybinder/cbkeybinder.h
===================================================================
--- a/src/plugins/contrib/keybinder/cbkeybinder.h	(revision 12011)
+++ b/src/plugins/contrib/keybinder/cbkeybinder.h	(revision 12012)
@@ -10,14 +10,6 @@
 #ifndef CBKEYBINDER_H
 #define CBKEYBINDER_H
 
-#if defined(__GNUG__) && !defined(__APPLE__)
-	#pragma interface "cbkeybinder.h"
-#endif
-
-#ifdef __BORLANDC__
-	#pragma hdrstop
-#endif
-
 #include <wx/string.h>
 //-#include <wx/timer.h>
 #include <wx/listbook.h>
Index: src/plugins/contrib/keybinder/keybinder.cpp
===================================================================
--- a/src/plugins/contrib/keybinder/keybinder.cpp	(revision 12011)
+++ b/src/plugins/contrib/keybinder/keybinder.cpp	(revision 12012)
@@ -12,11 +12,6 @@
 
 // Modified Keybinder for CodeBlocks KeyBnder v2.0 2019/04/8
 
-#ifdef __GNUG__
-#pragma implementation "keybinder.h" //necessary for linux, else undefines when linking
-#endif
-
-
 // includes
 #include <wx/event.h>
 #include <wx/frame.h> // Manager::Get()->GetAppWindow()
Index: src/plugins/contrib/keybinder/keybinder.h
===================================================================
--- a/src/plugins/contrib/keybinder/keybinder.h	(revision 12011)
+++ b/src/plugins/contrib/keybinder/keybinder.h	(revision 12012)
@@ -16,10 +16,6 @@
 #ifndef __KEYBINDER_G__
 #define __KEYBINDER_G__
 
-#ifdef __GNUG__
-#pragma interface "keybinder.h"
-#endif
-
 // required includes
 #include "wx/panel.h"
 #include "wx/dialog.h"
Index: src/plugins/contrib/keybinder/menuutils.cpp
===================================================================
--- a/src/plugins/contrib/keybinder/menuutils.cpp	(revision 12011)
+++ b/src/plugins/contrib/keybinder/menuutils.cpp	(revision 12012)
@@ -10,17 +10,10 @@
 // RCS-ID:      $Id$
 
 // menuutils for KeyBinder v2.0 2019/04/8
-#ifdef __GNUG__
-#pragma implementation "menuutils.h"
-#endif
 
 // For compilers that support precompilation, includes "wx/wx.h".
 #include "wx/wxprec.h"
 
-#ifdef __BORLANDC__
-#pragma hdrstop
-#endif
-
 #ifndef WX_PRECOMP
 #include "wx/wx.h"
 #endif
Index: src/plugins/contrib/keybinder/menuutils.h
===================================================================
--- a/src/plugins/contrib/keybinder/menuutils.h	(revision 12011)
+++ b/src/plugins/contrib/keybinder/menuutils.h	(revision 12012)
@@ -15,10 +15,6 @@
 #ifndef __WX_MENUUTILS_H__
 #define __WX_MENUUTILS_H__
 
-#ifdef __GNUG__
-#pragma interface "menuutils.h"
-#endif
-
 // includes
 #include "wx/panel.h"
 #include "wx/dialog.h"

------------------------------------------------------------------------
